test/modules/geo-layers/mvt-layer.spec.js

```diff
@@ -243,9 +245,42 @@ test('TileJSON', async t => {
       onAfterUpdate
     }
   ];
+
   await testLayerAsync({Layer: TestMVTLayer, viewport: testViewport, testCases, onError: t.notOk});
+
   t.end();
 
-  // restore fetcch
+  // restore fetch
   _global.fetch = fetch;
 });
+
+test('MVT#getVisibleTiles', async t => {
```

Comment:
I'd change this test to something more real. For example, set the viewport to the whole world and check you have the 4 expected tiles at z=1